### PR TITLE
Add a visual monitor.

### PR DESCRIPTION
**Pull request by mldk-tech**
#2

### CODE REVIEW

**Comment by mldk-tech** [comment]
@amitaibu - ^^

File: visual-monitor/test/tests.js [hunk 1]
```diff
@@ -21,17 +21,6 @@ var capsConfig = {
     'os' : 'Windows',
     'os_version' : '7',
     'resolution' : '1024x768'
-  },
-  'iphone5': {
```
Comment (amitaibu) [inline visual-monitor/test/tests.js:25]: btw, you can keep this - we just won't use it.

Comment (amitaibu) [inline visual-monitor/test/tests.js:25]: (for next time)

File: visual-monitor/test/tests.js [hunk 2]
```diff
@@ -58,15 +47,35 @@ describe('Visual monitor testing', function() {
...
+    client
+      .url('http://server2.web-stat.com/wixCurrencyPreview.pl')
+      .webdrivercss(testName + '.currency-preview', {
+        name: '1',
+        exclude:
+          [
+            // Currency text.
+            '#amt2',
+            // Currency result.
+            '#result',
```
Comment (amitaibu) [inline visual-monitor/test/tests.js:60]: can you exclude only the number 88.xx not the words (Euro ...)

Comment (mldk-tech) [inline visual-monitor/test/tests.js:60]: @amitaibu - Sorry, but no.
no

**Pull request merged**